pkg/operator/operator.go

```diff
@@ -24,28 +24,28 @@ const (
 	targtKubeAPIServerOperatorConfig = "instance"
 )
 
-type osinOperator struct {
+type authOperator struct {
 	configMap                   coreclientv1.ConfigMapsGetter
 	kubeAPIServerOperatorConfig dynamic.ResourceInterface
 }
 
-func NewOsinOperator(cmi v1.ConfigMapInformer, cm coreclientv1.ConfigMapsGetter, operatorConfigInformer controller.InformerGetter, operatorConfig dynamic.ResourceInterface) operator.Runner {
-	c := &osinOperator{
+func NewAuthenticationOperator(cmi v1.ConfigMapInformer, cm coreclientv1.ConfigMapsGetter, operatorConfigInformer controller.InformerGetter, operatorConfig dynamic.ResourceInterface) operator.Runner {
```

Comment:
Do not touch this file.  It works and keeps `kube:admin` functioning.  Please leave it alone.  We will delete it when we are ready.